_includes/links.html

```diff
@@ -1,2 +1,3 @@
 <a href="{{ "/about" | prepend: site.baseurl | replace: '//', '/' }}"><h2 class="header-link">About</h2></a>
-<a href="{{ "/atom.xml" | prepend: site.baseurl | replace: '//', '/' }}"><h2 class="header-link">RSS</h2></a>
\ No newline at end of file
+<a href="{{ "https://wiki.cybermouflons.com/" }}"><h2 class="header-link">Wiki</h2></a>
```

Comment:
```suggestion
<a href="https://wiki.cybermouflons.com"><h2 class="header-link">Wiki</h2></a>
```

No need to interpolation braces.
Also would be nice to remove the trailing slash :)